app/modules/settings/views/templates/display_settings.hbs

```diff
@@ -0,0 +1,35 @@
+<h2 class="layout-title">{{I18n 'display'}}</h2>
+
+<section class="entities">
+  <h3>{{i18n 'works'}} / {{i18n 'series'}} / {{i18n 'authors'}} {{i18n 'lists'}}</h3>
+  <a class="selector">
+    <span class="label">{{I18n 'current display'}} : </span>
+    <div class="{{#if entitiesCascade}}shown{{/if}}">{{icon 'th-large'}} {{I18n 'cascade'}}</div>
+    <div class="{{#if entitiesTable}}shown{{/if}}">{{icon 'align-justify'}} {{I18n 'table'}}</div>
+  </a>
+  <ul id="entitiesDisplayOptions">
+    <li displayType="entitiesDisplay" id="entitiesCascade">
+      <a>{{icon 'th-large'}} {{I18n 'select cascade display'}}</a>
+    </li>
+    <li displayType="entitiesDisplay" id="entitiesTable">
+      <a>{{icon 'align-justify'}} {{I18n 'select table display'}}</a>
+    </li>
+  </ul>
+</section>
+
+<section class="inventory">
+  <h3>{{i18n 'inventory lists'}}</h3>
+  <a class="selector">
+    <span class="label">{{I18n 'current display'}} : </span>
+    <div class="{{#if inventoryCascade}}shown{{/if}}">{{icon 'th-large'}} {{I18n 'cascade'}}</div>
+    <div class="{{#if inventoryTable}}shown{{/if}}">{{icon 'align-justify'}} {{I18n 'table'}}</div>
+  </a>
+  <ul id="inventoryDisplayOptions">
+    <li displayType="inventoryDisplay" id="inventoryCascade">
+      <a>{{icon 'th-large'}} {{I18n 'select cascade display'}}</a>
+    </li>
+    <li displayType="inventoryDisplay" id="inventoryTable">
+      <a>{{icon 'align-justify'}} {{I18n 'select table display'}}</a>
+    </li>
+  </ul>
```

Comment:
part of the reason my review on this PR was lagging is that this menu does not look very clear to me, and could be simplified (from the user point of view) by behaving like a `<select>` and having the selector and the information on the current state of that selector together (like the display selector in the inventory browser)

or even much simpler, an actual `<select>` (but then we loose the icons, but maybe that's ok for now as all this will be replaced by Svelte soon, which will make it trivial ;))
```suggestion
  <label for="inventoryDisplaySelector">{{i18n 'inventory lists'}}</label>
  <select id="inventoryDisplaySelector">
    <option value="inventoryCascade" {{#if inventoryCascade}}selected{{/if}}>{{I18n 'cascade'}}</option>
    <option value="inventoryTable" {{#if inventoryTable}}selected{{/if}}>{{I18n 'table'}}</option>
  </select>
```